CenterForOpenScience/osf.io
# Updating schema and adding management command. [ENG-2652]

## Purpose

Secondary schema needs updating, also adding in the ability for admins to manually update registration schemas without the need to run a migration.

## Changes

Changing secondary data schema. Adding in Jon's PR https://github.com/CenterForOpenScience/osf.io/pull/9448
## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify The schema is being updated with the management command in the admin app.
- Verify

What are the areas of risk? N/A

Any concerns/considerations/questions that development raised? N/A

## Documentation
N/A

## Side Effects
Shouldn't be 

## Ticket

https://github.com/CenterForOpenScience/osf.io/pull/9448
